packages/richtext-lexical/src/features/toolbars/shared/ToolbarDropdown/index.tsx

```diff
@@ -135,13 +135,6 @@ export const ToolbarDropdown = ({
     updateStates()
   }, [updateStates])
 
-  useEffect(() => {
-    document.addEventListener('mouseup', updateStates)
-    return () => {
-      document.removeEventListener('mouseup', updateStates)
-    }
-  }, [updateStates])
-
```

Comment:
I think the reason this was done is to update the toolbar when the editor loses focus. Any other changes to the content selection are handled in the `registerUpdateListener` below.

A much less aggressive and sufficient event would be an onBlur on the editor container. We could add a Lexical `BLUR_COMMAND` and set the selection to null, but arguably this is Lexical's default behavior so I've opened an issue there: https://github.com/facebook/lexical/issues/6622